aiida_crystal17/validation/schemas/inputd12.schema.json

```diff
@@ -365,6 +365,17 @@
                 "XXLGRID"
               ]
             },
+            "exact_exchange": {
+              "description": "the (pruned) integration grid",
```

Comment:
are these descriptions correct?

Reply:
Sorry, No they were not but they are now.